src/microbe_stage/editor/PatchMapNode.tscn

```diff
@@ -94,12 +94,11 @@ size_flags_vertical = 0
 mouse_filter = 2
 
 [node name="Eruption" type="TextureRect" parent="EventIcons"]
-visible = false
```

Comment:
This has again become visible...

Reply:
D'oh. Sorry. Didn't check the full changes to this file.

Reply:
Oh, I was actually expecting that this was Mr 42's PR, but I didn't actually read the actual author. So you probably wouldn't have seen that previous PR where I needed to multiple times comment on this same exact thing.